core/base/src/root-argparse.py

```diff
@@ -21,6 +21,6 @@ def get_argparse():
 	parser.add_argument('--web', help='Display graphics in a default web browser')
 	parser.add_argument('--web=<browser>', help='Display graphics in specified web browser')
 	parser.add_argument('[dir]', help='if dir is a valid directory cd to it before executing')
-	parser.add_argument('[file:data.root]', help='Open the ROOT file data.root')
-	parser.add_argument('[file1.C...fileN.C]', help='Execute the the ROOT macro file1.C ... fileN.C')
+	parser.add_argument('[data1.root...dataN.root]', help='Open the ROOT files data1.root ... dataN.root')
+	parser.add_argument('[file1.C...fileN.C]', help='Execute the the ROOT macro file1.C ... fileN.C.\nArguments can be passed to the macro with round parentheses after each filename.\nBetween filename and args, add "+" to precompile if needed\n"++" to force recompile, and "g" to include debug symbols')
```

Comment:
This should probably point to the manual (maybe implicitly) instead of repeating the documentation here.

Reply:
The drawback would be that this info would be not available when you travel (planes, etc.).

What about contracting it too:
`[+|++][g](args)`
and leaving out the explanation?

Reply:
You can't call `root foo.C(12)` in about any shell out there - this needs to be quoted, and users will get lost and annoyed if we don't show how to do this right. Then there's also `k` and `O`. The documentation will become as long as what we have in the manual (I hope!).

That's just my opinion, I'm happy to be overruled :-)

Reply:
`root foo.C+(12)` works well hehe, but true, not without the `+`.

Reply:
What are k and O doing? I just see g here: https://root.cern/manual/root_macros_and_shared_libraries/

Reply:
Oh, I see, it's in https://root.cern/doc/master/classTSystem.html#ac557d8f24d067a9b89d2b8fb261d7e18

Reply:
I have simplified it now and pointed to the manual.